examples/requirements.txt

```
@@ -0,0 +1,2 @@
+pytorch-lightning-bolts
```

```suggestion
lightning-bolts
```

We need to kill the other repo, was before my time unsure why it changed! this is the repo it should be I think